connectivity/FEATURE_BLE/source/cordio/TESTS/cordio_hci/driver/CMakeLists.txt

```diff
@@ -11,6 +11,11 @@ include(${MBED_PATH}/tools/cmake/mbed_greentea.cmake)
 project(${TEST_TARGET})
 
 mbed_greentea_add_test(
-    TEST_NAME ${TEST_TARGET}
-    TEST_REQUIRED_LIBS mbed-ble mbed-events
+    TEST_NAME
```

Comment:
The style change - was it done by a tool or just follow CMake docs style?

Reply:
Was done to follow style pointers from Lingkai for previous commits

Reply:
Possibly a bit of an over-interpretation - is it best as it was?

Reply:
This is the kind of indentation we have when calling a macro function with parameters, so it looks good to me. I guess what @0xc0170 means is whether it was done manually or by some tool, and the former is the case.

Reply:
We should set-up `cmake-format` as a pre-commit check and specify the style we want to use. It would be great to run the `cmake-format` check as part of the CI. That way we will auto-enforce the style and it will always be consistent. We can add a new issue to do this work in a separate PR.